<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<?pydev-testrun version="1.0"?><pydev-testsuite>
<summary errors="1" failures="0" finished="true" name="Test Run: 29" tests="1" total_time="Finished in: 60.75 secs."/>
<test location="C:\eclipse\workspace\Test_call.super.com\admin.py" status="error" test="test_firstPage.test_tearDown" time="">
<stdout><![CDATA[&#30331;&#24405;&#36229;&#32423;&#31649;&#29702;&#21592;&#25104;&#21151;
&#39318;&#39029;&#27979;&#35797;&#36890;&#36807;
&#25253;&#34920;&#31649;&#29702;&#8212;&#8212;&#22352;&#24109;&#35805;&#21153;&#25253;&#34920;&#8212;&#8212;&#26597;&#35810;&#25490;&#24207;&#23436;&#25104;
&#25253;&#34920;&#31649;&#29702;&#8212;&#8212;&#22352;&#24109;&#35805;&#21153;&#25253;&#34920;&#8212;&#8212;&#23548;&#20986;&#25104;&#21151;
&#25253;&#34920;&#31649;&#29702;&#8212;&#8212;&#22352;&#24109;&#35805;&#21153;&#25253;&#34920;&#8212;&#8212;&#26597;&#30475;&#19979;&#27454;&#35814;&#24773;&#36890;&#36807;
&#25253;&#34920;&#31649;&#29702;&#8212;&#8212;&#22352;&#24109;&#35805;&#21153;&#25253;&#34920;&#27979;&#35797;&#36890;&#36807;&#65281;
]]></stdout>
<stderr><![CDATA[Traceback (most recent call last):
  File "C:\eclipse\workspace\Test_call.super.com\admin.py", line 201, in setUp
    view_button = element('//*[@id="app"]/div/section/section/main/div/div/div/div/div[2]/table/tr[2]/td[10]/span')
  File "C:\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File "C:\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 978, in find_element
    'value': value})['value']
  File "C:\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "C:\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"xpath","selector":"//*[@id="app"]/div/section/section/main/div/div/div/div/div[2]/table/tr[2]/td[10]/span"}
  (Session info: chrome=73.0.3683.103)
  (Driver info: chromedriver=2.40.565498 (ea082db3280dd6843ebfb08a625e3eb905c4f5ab),platform=Windows NT 10.0.17134 x86_64)

]]></stderr>
</test>
<launch mode="debug">
<launch_memento><![CDATA[<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<launchConfiguration local="true" path="Test_call.super.com admin.py"/>
]]></launch_memento>
</launch>
</pydev-testsuite>
